docs/usage/firmware.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Firmware Updates
+weight: 70
+---
+
+Flatcar OS updates include the Linux firmware package which contains firmware binaries for those firmwares that get loaded at runtime.
+Some devices, however, need their firmware updates flashed out of band and may even require reboots for the updates to take effect.
+The vendors of those devices supply a firmware binary and instructions how to flash it.
+
+For example, the update utility may be a static binary and a specific firmware file for your server.
+These would need to be downloaded to the node and directly executed there through SSH.
+One could also automate the firmware updates a bit more on a Kubernetes cluster with a special firmware update container that runs on each node.
+
+Since the firmware updates are vendor-specific, there are no generic instructions on how to do them.
+To improve this situation and provide a unified and automated experience, the [Linux Vendor Firmware Service](https://fwupd.org/) was started.
+The system service `fwupd` can fetch firmware updates and flash them for supported devices.
+Not all vendors and not all devices are part yet of this effort but it is the only generic mechanism that can be documented as an example.
+
+## Firmware Update Container with fwupd
```

Comment:
This would be a lot more easier to follow for the user if the instructions were mentioned as steps.

Reply:
done